ieeeuoft/hackathon-template
# IEEE-193: pending order table formatting fixes

## Overview

- Resolves #IEEE-193 (partially)
- Fixed Issues from IEEE-193-create-pending-order-table:
The table width is different for each table ✅
The cells are all fixed widths (I did use styles.nowrap from a different table css file but i removed that for now) ✅
The buttons at the bottom are meant to be right aligned but it does not appear to be so, it is currently hovering weirdly in the center ✅
The Qty Granted field needs two elements, but their vertical alignments are not matching as I could not find a way to center align items vertically ✅

<img width="910" alt="image" src="https://user-images.githubusercontent.com/20618460/175833082-dea878b1-c549-41d3-bf52-e8fd78923dde.png">

## Unit Tests Created

- none, purely formatting PR


## Steps to QA

- go to localhost:3000/teams/<any team code> and look at the order tables
- play around with different screen sizes and see if anything looks funky
